src/java/org/apache/cassandra/schema/KeyspaceMetadata.java

```diff
@@ -62,6 +62,12 @@ public final class KeyspaceMetadata implements SchemaElement
 {
     public static final Serializer serializer = new Serializer();
 
+    public static boolean isValidKeyspaceName(String keyspaceName)
```

Comment:
@k-rus I would move this method to SchemaConstants.

Reply:
I replied in https://github.com/apache/cassandra/pull/4038#discussion_r2061246845 why I would prefer not to move. Let's see what you think about my reasoning.